_pages/en/strategy-build-vs-buy.md

```diff
@@ -241,37 +331,37 @@ Whether the solution is written by a team within the Government of Canada, or by
 For further guidance on the use of Open Source, please reference to Annex 5, Use of Open Source.
 
-##### 5. Minimize the size, cost, and scale of work packages
+##### 5. Minimize the size, cost, and scale of work packages<!-- omit in toc -->
 
-As discussed above, by increasing build decisions the government retains control over its ability to deliver on its mandate and removes context switches between organizations, resulting in system expertise leaving the government.
+As discussed above, by increasing build decisions the government retains control over its ability to deliver on its mandate and removes context switches between organizations, resulting in minimizing system expertise leaving the government.
 In order to further reduce the risk of build decisions, small incremental projects are advised over large ones.
-By decreasing the size of projects we decrease risk by increasing the likelihood of success.
+By decreasing the size of projects we decrease risk by increasing the likelihood of success (see [Target Solution Delivery Model](strategy-target-solution-delivery-model.html)).
 
 Further, this method of delivery ensures business value is provided as early as possible.
 If no business value is provided, the project may be cancelled without having dedicated the total budget, or entire time frame initially intended for its development.
 This approach also aligns with best practices for modern software delivery[^15]<sup>,</sup>[^16].
 This paradigm removes the need for large scale and risky projects.
-Instead, using a modular approach to software design we are able to reduce the likelihood of failure[^17], maintain agility to avoid vendor lock in, increase the vendor pool availability, and increase our adherence to direction from TBS by maintaining more control over the design decisions and implementation details of the solutions created on behalf of the government.
+Instead, using a modular approach to software design it is possible to reduce the likelihood of failure[^17], maintain agility to avoid vendor lock in, increase the vendor pool availability, and increase our adherence to direction from TBS by maintaining more control over the design decisions and implementation details of the solutions created on behalf of the government.
 As such, it is advised that when making build decisions, efforts be made for any given contract to be strictly less than $1 million, and any project is expected to have products able to deliver business value in strictly less than six months, ideally within weeks.
 Contracts strictly greater than $10 million should be exception cases for particularly complex systems[^18].
 
 For further discussion, please reference [Annex 1](#annex-1-context), specifically, [GC IT Project Analysis](#gc-it-project-analysis).
 
-### Next Steps
+### Next Steps<!-- omit in toc -->
```

Comment:
remove from Appendix A (diagnostic). This appendix should only list problem statements (the diagnosis) that is leading us to make policy decisions (the strategy).

Next Steps are the action plan items and the approval of the strategy